pkg/logs/schedulers/cca/scheduler.go

```diff
@@ -0,0 +1,81 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package cca
```

Comment:
NIT: I feel like `cca` is not a very discoverable acronym - personally I don't mind being more verbose with things like this. But that's just me. Feel free to ignore.

Reply:
I _think_ that will go away when container_collect_all support moves to AD, so I'll leave it for now.  But, thanks for calling it out -- it's a good point!